src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --button-color: #00acdc;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408vh;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+img {
+  width: 160px;
+  height: 134px;
+  top: 33px;
+  left: 20px;
+}
+
+.card__t {
+  width: 166px;
+  height: 36px;
+  margin-top: 40px;
+  margin-bottom: 4px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+  color: #060b35;
+}
+
+.card__c {
+  width: 98px;
+  height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__category {
+  box-sizing: border-box;
+  margin: 0;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__grade {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+  height: 16px;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(/images/star.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
The selector '.stars--4 :nth-child(-n + 4)' has a space between the class and pseudo-class which might not target the elements as intended. If you are trying to target the first four child elements of an element with class 'stars--4', you should remove the space.